src/Integration/LocalServices/RuleSetDeclaration.cs

```diff
@@ -17,13 +17,19 @@ namespace SonarLint.VisualStudio.Integration
     /// </summary>
     public class RuleSetDeclaration
     {
-        public RuleSetDeclaration(Property ruleSetProperty, string ruleSetPath, string activationContext, params string[] ruleSetDirectories)
+        public RuleSetDeclaration(Project project, Property ruleSetProperty, string ruleSetPath, string activationContext, params string[] ruleSetDirectories)
```

Comment:
I needed to project to be able to output the summary (per project/configurations).